### ISSUE TITLE
Add Rollbar

### ISSUE THREAD

**Issue by squaresurf** [spartansystems/raygun-rails-template#7]
We should have rollbar setup and ready to go.

Make sure to have the following in your config:

```ruby
config.exception_level_filters.merge!('ActionController::RoutingError' => 'ignore')
```

**Comment by samcdavid** [comment]
Do we really want that to be the default for all projects?

**Comment by squaresurf** [comment]
Absolutely. 99.9% of all routing errors will be false positives. We should always have google webmaster tools enabled that will allow us to catch the 0.01% of issues that are valid. By ignoring this, then we are increasing the value of rollbar communications which we could then hook into slack and maybe opsgenie.

**Comment by jasontwong** [comment]
I think this is okay is as long as everyone is using the proper rails link builders like users_path or posts_path. This only becomes an issue when we are manually linking to thinks with href's. Because yes, it's annoying to see routing errors for `/admin.php`

**Comment by squaresurf** [comment]
Also, how does a bad url get through if we're doing work to the `done done` state? If we're testing our work it is highly unlikely we'll add a valid error that would go into rollbar. Not to mention, the errors are so noisy in rollbar that it probably wouldn't get noticed there anyways.
